Task6/script.js

```diff
@@ -0,0 +1,33 @@
+const app = document.getElementById("js-app");
+const fragment = document.createDocumentFragment();
+
+const data = [
+  { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+];
+
+const fetchData = new Promise((resolve) => {
+  setTimeout(() => {
+    resolve(data);
+  }, 3000);
+});
+
+fetchData.then((value) => {
+  value.forEach((list) => {
```

Comment:
IMO
```suggestion
fetchData.then((values) => {
  values.forEach((value) => {
```

or

```suggestion
fetchData.then((items) => {
  items.forEach((item) => {
```

as you like.

Reply:
Thanks.
I understood that naming is easier to understand when looping if you write it in the plural first and then in the singular.